e2e/compatibility_test.go

```diff
@@ -0,0 +1,102 @@
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"gotest.tools/assert"
+
+	"gotest.tools/fs"
+	"gotest.tools/icmd"
+)
+
+func importAndTagImage(t *testing.T, cmd icmd.Cmd, tmpDir *fs.Dir, tag string, url string) string {
+
+	icmd.RunCmd(icmd.Command("wget", "-q", "-O", tmpDir.Join("img.tar"), url)).Assert(t, icmd.Success)
```

Comment:
You can use `icmd.RunCommand` instead (yes it's a mess, `icmd.RunCmd`, `icmd.Command`, `icmd.RunCommand`...)